### DISCUSSION TITLE
Add scalar to Golang

### DISCUSSION THREAD

**Discussion by MarceloPetrucio** [scalar/scalar#1627] [Ideas]
Hello everyone, how are you?

I noticed that there isn't a package focused on Go, so I created one to start supporting Go. The project is not 100% complete and does not have automated tests yet, but it gives an idea of how it can be done to be used with Go.

I made the package to be generic so it can be used with any Go framework.

Repository link: https://github.com/MarceloPetrucio/go-scalar-api-reference/

If you have any questions, I'm available to help.

I am happy to be able to help.

Best regards, Marcelo Petrucio.

**Comment by hanspagel** [comment]
You’re nuts! Great work.

**Reply by hanspagel** [reply]
We’ll add it to the root README. :) https://github.com/scalar/scalar/pull/1638

**Reply by MarceloPetrucio** [reply]
Wow, that's cool, I'm happy to have helped in some way. ;D
